packages/netlify-cms-core/src/components/Collection/__tests__/Sidebar.spec.js

```diff
@@ -20,6 +20,7 @@ jest.mock('../../../actions/collections');
 describe('Sidebar', () => {
   const props = {
     searchTerm: '',
+    isSearchEnabled: true,
```

Comment:
How about adding a new test with `isSearchEnabled: false`?

Reply:
Done in https://github.com/netlify/netlify-cms/pull/5509/commits/bc25bde9d105799834efa32531a0aded167b8577